api/agentpb/agent.proto

```diff
@@ -310,6 +310,8 @@ message CheckConnectionResponse {
   // Error message if connection check failed.
   string error = 1;
   Stats stats = 2;
+  // Contains the body of HTTP response of connection check to an exporter.
+  string exporter_response_body = 3;
```

Comment:
`CheckConnectionResponse` is about checking connectivity with Services, not with exporters. The word "exporter" is not used in this file at all.

Additionally, protobuf `string` can contain only UTF-8 data. I don't think we want limit us it it there

Reply:
@AlekSi, and what is your suggestion?

Reply:
https://github.com/percona/pmm/pull/689

Reply:
Thanks, elegant detour :)